### PR TITLE
Update README with LTS information

### PR DESCRIPTION
**Pull request by aggarw13**
Add text to informa bout LTS release of the **coreMQTT**, **coreHTTP**, **coreJSON**, **corePKCS11**, **backoffAlgorithm** and **AWS IoT Device Shadow client** libraries in the LTS release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,9 +197,9 @@ For example, a second release in June 2021 would be 202106.01. Although the SDK
 
 All of the released versions of the C-SDK libraries are available as git tags. For example, the last release of the v3 SDK version is available at [tag 3.1.2](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.1.2).
 
-### 202012.00
+### 202012.01
 
-[API documentation of 202012.00 release](https://docs.aws.amazon.com/embedded-csdk/202012.00/lib-ref/index.html)
+[API documentation of 202012.01 release](https://docs.aws.amazon.com/embedded-csdk/202012.00/lib-ref/index.html)
```
Comment (leegeth) [inline README.md:202]: Is the documentation still pointing to to 202012.00?

Comment (abhidixi11) [reply README.md:202]: Yes.

File: README.md [hunk 2]
```diff
@@ -59,6 +59,8 @@ The AWS IoT Device SDK for Embedded C (C-SDK) is a collection of C source files
 
 For the latest release of C-SDK, please see the section for [Releases](#releases).
 
+**C-SDK includes libraries that are part of the [FreeRTOS 202012.00 LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202012.00_LTS) release. Learn more about the FreeRTOS 202012.00 LTS libraries at https://freertos.org/lts-libraries.html.**
```
Comment (jainamt) [inline README.md:62]: pls confirm https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202012.00_LTS is the right link with Joe.

Comment (jainamt) [reply README.md:62]: the right link is https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202012.00-LTS

File: README.md (hunk 1 above)
Comment (jainamt) [reply README.md:202]: Yes, this was confirmed with Tanmoy

**Review by jainamt — APPROVED** [review]

**Review by abhidixi11 — DISMISSED** [review]

**Review by leegeth — DISMISSED** [review]

**Review by yourslab — APPROVED** [review]

**Review by leegeth — APPROVED** [review]

**Review by muneebahmed10 — APPROVED** [review]

**Review by jainamt — APPROVED** [review]

**Review by leegeth — APPROVED** [review]

**Review by jainamt — APPROVED** [review]

**Review by abhidixi11 — APPROVED** [review]

**Pull request merged**